### PR TITLE
fixed code to allow curators to update/unset a license added by a cur…

### PR DESCRIPTION
**Pull request by sweng66**
…ator

### CODE REVIEW

File: src/components/biblio/BiblioFileManagement.js [hunk 1]
```diff
@@ -338,7 +384,6 @@ const FileEditor = () => {
                             onChange={(event) => patchReferencefile(referenceFile.referencefile_id, {"pdf_type": event.target.value === "" ? null : event.target.value}, accessToken)}>
                 <option></option>
                 <option>ocr</option>
```
Comment (azurebrd) [inline src/components/biblio/BiblioFileManagement.js:386]: @sweng66 This tif option shouldn't be removed.  It probably happened from some merge oversight ?

**Review by azurebrd — CHANGES_REQUESTED** [review]
Look great @sweng66 !  (just put back the "tif" <option> that shouldn't be removed, thanks !)

File: src/components/biblio/BiblioFileManagement.js (hunk 1 above)
Comment (sweng66) [reply src/components/biblio/BiblioFileManagement.js]: Great catch!! Thanks @azurebrd 

Added back

**Comment by azurebrd** [comment]
Great, thanks @sweng66 !

**Pull request merged**